wire-ios-data-model/Source/Model/Conversation/Actions/SyncUsersAction.swift

```diff
@@ -0,0 +1,49 @@
+////
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
```

Comment:
```suggestion
// Copyright (C) 2024 Wire Swiss GmbH
```

Reply:
Question: should we regularly update the Year in all files? It seems a bit of a flooding and until now I don't get the benefit...

Reply:
I am not sure, I am not up to date how we should update them. I think it's when you edit them you should update